packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -261,6 +261,16 @@ const defaultConfigs = {
     type: "uniswap",
     uniswapAddress: "0x683ea972ffa19b7bad6d6be0440e0a8465dba71c",
     twapLength: 7200
+  },
+  BTCDOM: {
+    type: "domfi",
+    pair: "BTCDOM",
+    minTimeBetweenUpdates: 60
+  },
+  ALTDOM: {
+    type: "domfi",
+    pair: "ALTDOM",
+    minTimeBetweenUpdates: 60
```

Comment:
How come no lookback in the default even though it's a required param?

Reply:
That's my bad, I must have accidentally removed it earlier, I realize `lookback` became `twapLength` in the others...
Anyhow, I've fixed it 👍🏾